### PR TITLE
Homepage converted to react from html

### PR DESCRIPTION
**Pull request by surbhi84**
please ignore the links most of them might be broken, i will add them once...more pages get created. This is just a conversion of html to react.

### CODE REVIEW

**Review by prakashsakari — APPROVED** [review]
1. Files structure looks good
2. Just a suggestion - add a screenshot of the UI as well.

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+/public/assets
```
Comment (SanketDhabarde) [inline .gitignore:3]: Don't add .gitignore to pr going forward.

File: src/backend/db/users.js [hunk 2]
```diff
@@ -17,3 +17,8 @@ export const users = [
     updatedAt: formatDate(),
   },
 ];
+
+/*
```
Comment (SanketDhabarde) [inline src/backend/db/users.js:21]: You can remove commented code.

File: src/components/carousel/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import carousel from "./carousel.module.css";
+import { useState } from "react";
+import { v4 as uuid } from "uuid";
+
+function SliderLabels({ activeSlide, setActiveSlide, inputNum }) {
```
Comment (SanketDhabarde) [inline src/components/carousel/Carousel.jsx:5]: You can move it to a separate file

File: src/components/carousel/carousel.module.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  position: relative;
+}
+
+.slides {
+  width: 400%;
+  height: 100vh;
+  display: flex;
+}
+
+/* .slides input {
```
Comment (SanketDhabarde) [inline src/components/carousel/carousel.module.css]: Remove commented code

File: src/components/carousel/carousel.module.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
+/* css for carousel below */
```
Comment (SanketDhabarde) [inline src/components/carousel/carousel.module.css:1]: Just wanted to know. what is the use of `module.css` extension?

File: src/components/carousel/carousel.module.css [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  padding: 0.3rem;
+  border-radius: 50%;
+  cursor: pointer;
+  transition: 1s;
+}
+
+.manual-btn:hover {
+  background: var(--primary-color);
+}
+/*
```
Comment (SanketDhabarde) [inline src/components/carousel/carousel.module.css]: Remove this as well

File: src/css/common.css [hunk 7]
```diff
@@ -0,0 +1,118 @@
+@import url(https://halloui.netlify.app/halloUi-share.css);
+/* @import url(/css/homepage.css);
+@import url(/css/header.css);
+@import url(/css/footer.css);
+@import url(/css/variable.css);
+@import url(/css/util.css);
+@import url(/pages/productList/productList.css);
+@import url(/pages/wishlist/wishlist.css); */
```
Comment (SanketDhabarde) [inline src/css/common.css:8]: Remove this as well if not using

File: yarn.lock [hunk 8]
```diff
@@ -1,11662 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (SanketDhabarde) [inline yarn.lock:1]: No need to add this file as well.

**Review by SanketDhabarde — APPROVED** [review]
Just left a few suggestions. everything else looks good. 
- I like the code structure
- You can put images in public/images will look good

File: yarn.lock (hunk 8 above)
Comment (surbhi84) [reply yarn.lock:1]: i have put it in git ignore after this pr

File: src/css/common.css (hunk 7 above)
Comment (surbhi84) [reply src/css/common.css:8]: i just left them in case i'll need the refrence in future.

File: src/components/carousel/carousel.module.css (hunk 5 above)
Comment (surbhi84) [reply src/components/carousel/carousel.module.css:1]: The css modules are used to provide scope to the css files, this way redundant classnames in different css files doesnt overrride each other.

**Pull request merged**